### DISCUSSION TITLE
Is it ok when debug stuck if WSL is used and "externalConsole": true

### DISCUSSION THREAD

**Discussion by yh-sb** [microsoft/vscode-cpptools#8246] [Q&A]
I have just found that you can't start debug under WSL if `"externalConsole": true` is set in launch.json.

**Is it known limitation or bug?**

*Information about my system:
WSL 2, Windows 11 22000.194, vscode 1.60.1, vscode-cpptools 1.7.0-insiders2*

**Comment by sean-mcmanus** [comment]
@WardenGnaw might be able to answer on Monday.

**Comment by yh-sb** [comment]
Still reproduced. Created the issue: https://github.com/microsoft/vscode-cpptools/issues/11418
